packages/editor/src/panels/files/helpers.tsx

```diff
@@ -94,6 +96,14 @@ export const CurrentFilesQueryProvider = ({ children }: { children?: ReactNode }
     filesState.searchText.value
   )
 
+  const uploadsQuery = useFind(fileBrowserUploadPath, {
```

Comment:
this does not do anything - fileBrowserUploadPath does not have a find function implemented